ui/pages/settings.js

```diff
@@ -19,7 +20,9 @@ class Settings extends React.Component {
         <Head>
           <title>Settings | Meshery</title>
         </Head>
-        <MesherySettings />
+        <Paper>
```

Comment:
The sharp corners at the top of this paper aren't quite as nice as what we have today with rounded corners.

The text at the bottom of the paper is missing the padding that it has today.

Moving to Paper as a component might be the right thing to do. I'm supportive, if so. We'll need to ensure that we're not introducing regression issues in terms of style, though.

Reply:
My bad @leecalcote . Should have checked other pages before. Its fixed now :rocket: . Also fixed #1461

Reply:
Nice work. Looks good.